pentaho-i18n-bundle/pom.xml

```diff
@@ -16,12 +16,11 @@
   <dependencies>
     <dependency>
       <groupId>org.apache.felix</groupId>
-      <artifactId>org.apache.felix.framework</artifactId>
-      <version>${felix.version}</version>
+      <artifactId>org.apache.felix.fileinstall</artifactId>
```

Comment:
Didn't you conclude this fileinstall dependency was unused?

Reply:
I'll double check.